src/exceptions/LabellerException.class.php

```diff
@@ -1,16 +1,27 @@
 <?php
-require_once TRIPOD_DIR . '/exceptions/TripodException.class.php';
+
+namespace Tripod\Exceptions;
+
+require_once TRIPOD_DIR . '/exceptions/Exception.class.php';
 
 /**
  * @codeCoverageIgnore
  */
-class TripodLabellerException extends TripodException {
+class LabellerException extends Exception {
```

Comment:
Is this extending \Exception or \Tripod\Exceptions\Exception? I guess there is a right answer, but would naming make it clearer.